ansible/environments.yml

```diff
@@ -103,3 +103,18 @@
           image_name: "d324da4b-b827-44fb-8331-bfad41a28aca"
       tags:
         - dev2-env
+
+    - name: Ensure aws_dev environment stack
+      cloudformation:
+        stack_name: "ae-aws-dev"
+        state: "present"
+        region: "us-west-2"
+        disable_rollback: true
+        template: "../aws/ae-environment.yml"
+      args:
+        template_parameters:
+          EnvName: "dev_aws"
+          ApiPort: 3013
+          NodeCount: 2
+          AmiId: "ami-34a9364c" # AMI  ubuntu 16.04 with master user + python added.
```

Comment:
Is this 16.04.3?
Is this publicly built, or does "with master user + python added" mean it is custom built by us?

Reply:
public image, with added master user and python.